resources/views/teachingObject/create.blade.php

```diff
@@ -84,6 +84,18 @@
                             </div>
                         </div>
 
+                        <div class="form-group">
+                          <label for="selectTags" class="col-md-4 control-label">Tags</label>
```

Comment:
aca cambiembosle el nombre de for. Pongamosle solo tags (en plural). Lo mismo en el name del select y el id del select. Esto te va a impactar tambien en el codigo del controller.